logrhythmnextgensiem.json

```diff
@@ -0,0 +1,987 @@
+            "order": 3
+        },
+        "first_scheduled_ingestion_span": {
+            "description": "Poll last n days for first scheduled polling",
+            "default": 10,
+            "required": true,
+            "data_type": "numeric",
+            "order": 4
+        },
+        "max_alarms": {
+            "description": "Maximum number of alarms to ingest for scheduled polling",
+            "data_type": "numeric",
+            "default": 100,
+            "required": true,
+            "order": 5
+        }
+    },
+    "actions": [
+        {
+            "action": "test connectivity",
+            "identifier": "test_connectivity",
+            "description": "Validate the asset configuration for connectivity using supplied configuration",
+            "type": "test",
+            "read_only": true,
+            "parameters": {},
+            "output": [],
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "on poll",
+            "identifier": "on_poll",
+            "description": "Ingest alarms from LogRhythm",
+            "type": "ingest",
+            "read_only": true,
+            "parameters": {
+                "container_id": {
+                    "description": "Parameter ignored in this app",
+                    "data_type": "numeric"
+                },
+                "start_time": {
+                    "description": "Parameter ignored in this app",
+                    "data_type": "numeric"
+                },
+                "end_time": {
+                    "description": "Parameter ignored in this app",
+                    "data_type": "numeric"
+                },
+                "container_count": {
+                    "description": "Maximum number of container records to query for",
+                    "data_type": "numeric"
+                },
+                "artifact_count": {
+                    "description": "Parameter ignored in this app",
+                    "data_type": "numeric"
+                }
+            },
+            "output": [],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "update alarm",
+            "identifier": "update_alarm",
+            "description": "Update an alarm",
+            "type": "generic",
+            "read_only": false,
+            "parameters": {
+                "id": {
+                    "description": "Alarm ID",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": true,
+                    "contains": [
+                        "logrhythm alarm id"
+                    ],
+                    "order": 0
+                },
+                "status": {
+                    "description": "New status for the alarm",
+                    "data_type": "string",
+                    "value_list": [
+                        "",
```

Comment:
Confirmation: Do we want to allow empty string here?

Reply:
Done